errors_113.go

```diff
@@ -0,0 +1,70 @@
+// This file implements the helper interfaces for the errors package introduced in Go 1.13
+// +build go1.13
+
+package errorx
+
+import "errors"
+
+// As implements the helper interface for errors.As.
+// If v is a pointer to a pointer to Error, *v will be set to this error.
+// Otherwise, it traverses the error chain until it reaches a non-*Error error and calls errors.As on that error and the
+// target.
+func (e *Error) As(v interface{}) bool {
+	if target, ok := v.(**Error); ok {
```

Comment:
I'm not sure we only have to check for go type. If it's an `errorx` error we're talking about, shouldn't we also check its `.Type()`?

Reply:
I don't think so. `As` wouldn't check `.Type()` because it's looking if you can assign `v` to the go type and sets it if you can, otherwise it continues following the chain until it finds something that _is_ assignable.

https://golang.org/pkg/errors/#As

I don't think transparency really plays a role in `As` since it's really just a way to cast a generic `error` into a specific type that you can inspect its values for?  In fact, I'm not even sure we really need to define an `As` method since we have an `Unwrap` and `errors.As` will just keep calling `Unwrap`.

Reply:
Ok, I think I see what's going on, I was focused on transparency but it sounds like everything should just stop if `!currentErr.transparent`.

Reply:
That's why I said I'm not sure. errorx defines its own "error type" system based on a single go type, so it is worth considering that we could maintain this in `.As()` method.
Perhaps you're right, and right now we should just remove this implementation and rely on `Unwrap()`